vignettes/estimate_infections_workflow.Rmd

```diff
@@ -117,6 +127,19 @@ Gamma(mean = Normal(3, 2), sd = Normal(1, 0.1), max = 10)
 #>         1.4
 ```
 
+which looks like this when plotted
+
+
+``` r
+suppressWarnings(
+  plot(
+    Gamma(mean = Normal(3, 2), sd = Normal(1, 0.1), max = 10)
+  )
+)
```

Comment:
I would prefer to assign this to a variable earlier and then here can just plot the variable (without requiring suppressWarnings).

Reply:
See 0cda4404a6a3489f10fee7a25a87139747f8c86b